libsrc/c128/emd/c128-reu.s

```diff
@@ -92,17 +92,56 @@ INSTALL:
         cmp     REU_REUADDR             ; Check for presence of REU
         bne     nodevice
 
-        ldy     #>(128*4)               ; Assume 128KB
-        lda     REU_STATUS
-        and     #$10                    ; Check size bit
-        beq     @L1
-        ldy     #>(256*4)               ; 256KB when size bit is set
-@L1:    sty     pagecount+1
-
```

Comment:
this is good - the "size" bit isnt actually related to size at all, its the status of the jumper that tells the REC chip what kind of RAM is installed